README.md

```diff
@@ -59,6 +59,301 @@ drwxr-xr-x  2 you  staff    68 Mar 29 14:16 .cache
 ```
 *Security notice:* the file `.netrc`, created or updated by `conjur authn login`, contains a user identity credential that can be used to access the Conjur API. You should remove it after use or otherwise secure it like you would another netrc file.
 
+## Usage
+
+```
+NAME
+    conjur - Command-line toolkit for managing roles, resources and privileges
+
+SYNOPSIS
+    conjur [global options] command [command options] [arguments...]
+
+GLOBAL OPTIONS
+    --help    - Show this message
+    --version - Display the program version
+```
+
+### Commands
+
+[authn](#authn)       - Login and logout
```

Comment:
You could use a markdown table for the TOC here - it might look much better and be more condensed

Reply:
Good idea